src/roman_datamodels/stnode/_node.py

```diff
@@ -10,10 +10,30 @@
 from asdf.lazy_nodes import AsdfDictNode, AsdfListNode
 from asdf.tags.core import ndarray
 from astropy.time import Time
+from semantic_version import Version
 
-from ._registry import SCALAR_NODE_CLASSES_BY_KEY
+from ._registry import INTERNAL_WRAP_LIMITS, SCALAR_NODE_CLASSES_BY_KEY
 
-__all__ = ["DNode", "LNode"]
+__all__ = ["DNode", "LNode", "TaggedScalarDNode"]
+
+
+def _convert_to_scalar(key, value, ref=None):
+    """Wrap scalars in into a tagged scalar object"""
+    from ._tagged import TaggedScalarNode
+
+    if isinstance(ref, TaggedScalarNode):
+        # we want the exact class (not possible subclasses)
+        if type(value) == type(ref):  # noqa: E721
```

Comment:
```suggestion
        if type(value) is type(ref):
```
Does `is` work here and allow the noqa removal?

Reply:
I think so, good catch! I was just copying existing code around and into a new function.